File: menu.py
import os
def printMenu():
    ''' printMenu
    Author: Santillan Osorio Ada Paulina
    Date: 10/03/2023
    Description: Descripción de los tipos de datos en python
    '''
    os.system('cls')
    print("Bienvenido al menú seleccione una opciónn \n\n")
    print("\t1)SUMA\n")
    print("\t2)RESTA\n")
    print("\t3)MULTIPLICACION\n")
    print("\t4)DIVISION\n")
    option = input()
    if(option):
        return int (option)
    else:
        return 0


def main():
    flag = 0
    inutil =''
    while(flag!=4):
        flag = printMenu()
        
        if (flag==1):
            print("\n1 SUMA")
            n1=int(input("Intro número uno: "))
            n2=int(input("Intro numero dos: "))
            suma=n1+n2
            print("La suma es: ",suma)
            if suma % 2 == 0:
                print(suma, "El número es par")
            else:
                print(suma,"El número es impar")
        elif (flag==2):
            print("\n2  RESTA")
            num1=int(input("Intro número uno: "))
            num2=int(input("Intro numero dos: "))
            resta=num1-num2
            if resta%5==0:
                print(resta,"El número es multiplo de 5")
            else:
                print(resta,"El numero es impar")
        elif (flag==3):
            print("\n3 MULTIPLICACION")
            numero1=int(input("Intro número uno: "))
            numero2=int(input("Intro numero dos: "))
            mult=numero1+numero2
            if mult>=10:
                print(mult,"Es mayor a 10")
            else:
                print(mult,"es menor a 10")
        elif (flag==4):
            print("\n4 DIVISION")
            enum1=int(input("Intro número uno: "))
            enum2=int(input("Intro numero dos: "))
            division=enum1-(enum1//enum2*enum2)
            if division == 0:
                print(division,"No tiene residuo")
            else:
                print(division,"Tiene residuo")
        elif (flag==0):
            print("\nNo valido")
        else:
            print("\nNo valido")

        print("\nOprima enter tecla cualquiera para continuar ")
        input(inutil)
    print("excelente")

if __name__ == "__main__":
    main()
    
 #1.¿Qué es lo que hace el código en un comentario antes de modificarlo?
#Nos muestra el menú y a la hora de seleccionar una opción nos parece “no valido” y a continuación de que oprimamos enter para continuar y nos mostrara de nuevo el menú.
#2.	¿Cuál es la sintaxis de un while?
#Un ciclo while evalúa una condición y ejecuta el código si la condición es verdadera. El bloque de código se ejecuta repetidamente hasta que la condición se vuelve falsa o se vuelve falsa.
#3.	¿Exite swhitch en python?
#No existe switch en python
#4.	¿Qué es un elif?
#Es como si combináramos else if, utilizada para enlazar varios else if y aumentar las tabulaciones.
#5.	¿Qué función de se ejecuta al correr el script?
#Sintaxis de comandos  

    

File: menu.c
/* printMenu
    Author: santillan Osorio Ada Paulina
    Date: <10/03/2023>
    */

#include <stdio.h>
#include <conio.h>
#include <stdlib.h>

int printMenu(){

    int option;
    system("clear");
    fflush( stdin );
    printf("Bienvenido al men%c seleccione una opci%cn \n\n",163,162);
    printf("\t1)Suma\n"); printf("\t2)Resta\n"); printf("\t3)Multiplicacion\n"); printf("\t4)Division\n");
    scanf("%d",&option);
    if(option)
        return option;
    else
        return 0;
}

int main(){
    char inutil;
    int flag = 0;
    while(flag!=4){
        flag = printMenu();
        switch (flag){
            case 1:
                printf("\n1 Suma");
                    int num1,num2,sumar;
                    printf( "\n   Introduzca primer numero: " );
                    scanf( "%d", &num1 );
                    printf( "\n   Introduzca segundo numero: " );
                    scanf( "%d", &num2 );
                    sumar = num1 + num2;
                    printf( "\n   La suma es: %d", sumar);
                    if (sumar % 2 == 0) {
                        printf("Es par");
                    } else { 
                        printf("Es impar"); }
                break;
            
            case 2:
                printf("\n2 Resta");
                int n1,n2,restar;
                    printf( "\n   Introduzca primer numero: " );
                    scanf( "%d", &n1 );
                    printf( "\n   Introduzca segundo numero: " );
                    scanf( "%d", &n2 );
                    restar = n1 - n2;
                    printf( "\n   La Resta es: %d", restar);
                    if (restar %5==0) {
                        printf("Es multiplo de 5");
                    } else { 
                        printf("Es impar"); }
                break;
            
            case 3:
                printf("\n3 Multiplicacion");
                int mult, enum1,enum2;
                    printf( "\n   Introduzca primer numero: " );
                    scanf( "%d", &enum1);
                    printf( "\n   Introduzca segundo numero: " );
                    scanf( "%d", &enum2 );
                    mult = enum1 * enum2;
                    printf( "\n   La Multiplicacion es: %d", mult);
                    if (mult >= 10 ) {
                        printf("Mayor a 10");
                    } else { 
                        printf("Menor a 10"); }
                break;
                
            case 4:
                printf("\n4 Division");
                float numero1,numero2,dividir;
                printf( "\n   Introduzca primer numero: " );
                scanf( "%f", &numero1 );
                printf( "\n   Introduzca segundo numero: " );
                scanf( "%f", &numero2 );
                dividir = numero1 / numero2;
                printf( "\n   La Division es: %f", dividir);
                break;
            
            case 0:
                printf("\nNo valido");
                break;
            default:
                printf("\nNo valido");
                break;
        }
        printf("\nOprima enter tecla cualquiera para continuar ");
        fflush( stdin );
        scanf("%c",&inutil);
    }
    printf("excelente");
}
